neo4j-jdbc-bolt/src/test/java/org/neo4j/jdbc/bolt/utils/JdbcConnectionTestUtils.java

```diff
@@ -130,9 +130,6 @@ public static void closeConnection(Connection connection, Statement stmt, Result
             }
         } catch (SQLException e) {
             e.printStackTrace();
-        } finally {
```

Comment:
just a question. So if I understood correctly we can get rid of this because we now inject the connection via factories skipping all the cache management right?

Reply:
Exactly, the cache is only used in integration tests now